brainstormforce/ultimate-addons-for-gutenberg
# Icon List - Acceptance Testing Round 2

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/560b03e7-3835-4fd6-8994-487ca07a1b4a?redirect=true&view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Screenshots

- Screencast added in task comment.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Icon list.
- Check after Removing the label is icon list editable or not.
- Check the Responsive option of the Background in the responsive device frontend & editor as well.

### Checklist:
